packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -491,6 +491,31 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     [/* @conditional-compile-remove(rich-text-editor-image-upload) */ handleInlineImageUploadAction, messageThreadProps]
   );
 
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  const removeImageTags = useCallback((event: { content: DocumentFragment }) => {
+    event.content.querySelectorAll('img').forEach((image) => {
```

Comment:
This code is reused between CopyPastePlugin and ChatScreen components. Let's move it out to the common package